packages/@stimulus/core/src/application.ts

```diff
@@ -28,13 +29,17 @@ export class Application implements ErrorHandler {
 
   async start() {
     await domReady()
+    this.logDebugActivity("application", "starting")
     this.dispatcher.start()
     this.router.start()
+    this.logDebugActivity("application", "started")
```

Comment:
All other debug logs use the infinite tense (`initialize`, `connect`), not here.
Maybe we can harmonize and just do `this.logDebugActivity("application", "start")`